### PR TITLE
[ONNX] opset 11 updates

### PR DESCRIPTION
**Pull request by neginraoof**
Opset 11 updates:
- Enabled ORT tests for updated ops in opset 11
- Updated index_copy and index_fill symbolic for opset 11 to modify onnx::Scatter -> onnx::ScatterElemets

### CODE REVIEW

**Comment by neginraoof** [comment]
cc @houseroad for review

File: torch/onnx/symbolic_opset11.py [hunk 1]
```diff
@@ -97,7 +98,7 @@ def scatter(g, self, dim, index, src):
 
 @parse_args('v', 'i', 'none')
 def cumsum(g, self, dim, dtype=None):
-    dim_tensor = g.op("Constant", value_t=torch.tensor(dim))
+    dim_tensor = g.op("Constant", value_t=torch.tensor(dim, dtype=torch.int))
```
Comment (BowenBao) [inline torch/onnx/symbolic_opset11.py]: why this change?

**Review by BowenBao — APPROVED** [review]
LGTM

File: torch/onnx/symbolic_opset11.py (hunk 1 above)
Comment (neginraoof) [reply torch/onnx/symbolic_opset11.py]: ONNX spec here, it only takes integers

**Comment by neginraoof** [comment]
cc @houseroad for review

**Review by houseroad — APPROVED** [review]
Looks good.
